File: sqlparser/src/sql/csgen/SqlXmlStatement.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;
namespace CData.Sql {


public class SqlXmlStatement : SqlStatement{
  string xml;
  
  public SqlXmlStatement(string xmlString, Dialect dialectProcessor) : base(dialectProcessor) {
    xml = xmlString;
  }

  private SqlXmlStatement(Dialect dialectProcessor) : base(dialectProcessor) {
  }
  
  public string GetXmlString(){
    return xml;
  }

  public override void Accept(ISqlQueryVisitor visitor) {
  }

  public override Object Clone() {
    SqlXmlStatement obj = new SqlXmlStatement(null);
    obj.Copy(this);
    return obj;
  }

  protected override void Copy(SqlStatement obj) {
    base.Copy(obj);
    SqlXmlStatement o = (SqlXmlStatement)obj;
    this.xml = o.xml;
  }
}
}

File: sqlparser/src/sql/SqlXmlStatement.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;
namespace CData.Sql {
#*/

public class SqlXmlStatement extends SqlStatement{
  String xml;
  
  public SqlXmlStatement(String xmlString, Dialect dialectProcessor) {
    super(dialectProcessor);
    xml = xmlString;
  }

  private SqlXmlStatement(Dialect dialectProcessor) {
    super(dialectProcessor);
  }
  
  public String getXmlString(){
    return xml;
  }

  public /*#override#*/ void accept(ISqlQueryVisitor visitor) throws Exception {
  }

  @Override
  public Object clone() {
    SqlXmlStatement obj = new SqlXmlStatement(null);
    obj.copy(this);
    return obj;
  }

  @Override
  protected void copy(SqlStatement obj) {
    super.copy(obj);
    SqlXmlStatement o = (SqlXmlStatement)obj;
    this.xml = o.xml;
  }
}
